README.md

```diff
@@ -140,12 +140,28 @@ description: 'Bindings to SQLite'
     <td>
 
 ```yaml
-compiler-opts: '-I/usr/lib/llvm-9/include/'
+compiler-opts: # You can also pass in a string.
```

Comment:
`# You can also pass in a string.`

Maybe just remove this.

Reply:
Done.

I noticed I had set my CPATH variable but many users wouldn't have, so I've added this location `/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include`
in the include path as well.

These are the system headers that are only available in this location.